TestModels/LocalService/src/WrappedIndex.dfy

```diff
@@ -3,7 +3,7 @@
 
 include "../Model/SimpleLocalServiceTypesWrapped.dfy"
 
-module {:extern "simple.localservice.internaldafny.wrapped"} WrappedSimpleLocalService refines WrappedAbstractSimpleLocalServiceService
+module {:extern "simple.localservice.internaldafny.wrapped"} WrappedSimpleLocalServiceService refines WrappedAbstractSimpleLocalServiceService
```

Comment:
This will break other languages. Either check-in the shim manually with a TODO, or cut a SIM and add a TODO that we need to fix this in other languages too.

Reply:
I reverted it back but we should think of this as a clean up task. I added it in sim.